fe/fe-core/src/main/java/com/starrocks/common/Log4jConfig.java

```diff
@@ -309,7 +309,8 @@ private static String generateXmlConfTemplate() {
         }
 
         String newXmlConfTemplate = APPENDER_TEMPLATE;
-        newXmlConfTemplate += Config.sys_log_to_console ? CONSOLE_LOGGER_TEMPLATE : FILE_LOGGER_TEMPLATE;
+        newXmlConfTemplate += System.getenv("SYS_LOG_TO_CONSOLE").trim().equals("1")
```

Comment:
probably the configvar is still kept. but the default value can be changed via the env var. so end-user can still use fe.conf configvar to overwrite the config, and in container env, suppose the user doesn't have the configvar in fe.conf, our env var will overwrite the default value and turn on/off the log to console. What do you think?

Reply:
@kevincai, i feel having multiple ways to handle one behavior leads to confusion, rather we should have one method. either config or env variable. there can come situations where user may by mistake configure both, then which will take precedence, config or env var; one user may feel config should take precedence other may be env variable should take precedence. what is your opinion?

Reply:
the configvar is always the canonical one to control the behavior, the env var is just changing the default value of the configvar. if the configvar is explicitly set in fe.conf, then the default value will not be used.

Reply:
@kevincai i have updated as per our review discussion.